src/Avalonia.OpenGL/EglDisplay.cs

```diff
@@ -36,7 +36,7 @@ public EglDisplay(EglInterface egl, int platformType, IntPtr platformDisplay, in
                         throw new OpenGlException("eglGetPlatformDisplayEXT is not supported by libegl.dll");
 
                     var allowedApis = AvaloniaLocator.Current.GetService<AngleOptions>()?.AllowedPlatformApis
-                                      ?? new List<AngleOptions.PlatformApi> {AngleOptions.PlatformApi.DirectX9};
+                                      ?? new List<AngleOptions.PlatformApi> {AngleOptions.PlatformApi.DirectX11};
```

Comment:
Do we still want to allow D3D9 if D3D11 isn't available? Or do all the Windows' that we run on support both?

Reply:
Y we could just add DirectX9 as the second item

Reply:
Will that still fix it then?

Reply:
yes